content/master/concepts/pods.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Crossplane Pods
+weight: 1
+description: Background on the components installed with Crossplane and their functions.
+---
+
+The base Crossplane installation consists of two Pods, the `crossplane` pod and
+the `rbac-manager` pod. Both pods install in the `crossplane-system`
+namespace by default. 
+
+
+## Crossplane pod
+
+### Init pod
+Before starting the core Crossplane pod an _init_ Pod runs. The init pod is
+responsible for setting parameters related to the core Crossplane pod and
+installing the core Crossplane 
+[Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+(`CRDs`).
+
+The settings the init pod sets include installing Provider or Configuration 
+packages with Crossplane, customizing the namespace Crossplane installs in and 
+defining webhook configurations. 
+
+The core CRDs installed by the init pod include: 
+* Composite Resource Definitions, Compositions, Configurations and Providers
+* `locks` to manage package dependencies
+* `controllerconfigs` to apply settings to installed Providers
+* `storeconfigs` for connecting external secret stores like 
+[Hashicorp Vault](https://www.vaultproject.io/)
+
+{{< hint "note" >}}
+
+The [Install Crossplane]({{< ref "../software/install" >}}) section has more
+information about customizing the Crossplane install.
+{{< /hint >}}
+
+The init pod completes and starts the Crossplane core pod. 
+
+### Core pod
+
+The main Crossplane pod, called the _core_ pod, enforces
```

Comment:
It's getting confusing with the naming here. We refer to the main Crossplane pod as "core" pod, but its name is `crossplane` in the `crossplane-system` namespace, right? Not sure we can do any better than what's written but just calling it out.

Reply:
More accurately it's the init _container_ and core _container_. Both of these containers are in the `crossplane` _pod_ inside the `crossplane-system` namespace. The name "core" is what Crossplane calls it in source. 

Would being more specific that they are two _containers_ inside a single _pod_ help? Otherwise we're stuck with